test_post_request.py

```diff
@@ -15,15 +17,17 @@ def do_post_request():
     #                                              y=["leftamygdala"])
 
     # example request 2
-    algorithm_input_data = AlgorithmInputDataDTO(pathology="dementia",
-                                                 datasets=["demo_data"],
-                                                 filters=None,
-                                                 x=["lefthippocampus", "righthippocampus"],
-                                                 y=["alzheimerbroadcategory_bin"])
-
-    algorithm_request = AlgorithmRequestDTO(inputdata=algorithm_input_data,
-                                            parameters=None,
-                                            crossvalidation=None)
+    algorithm_input_data = AlgorithmInputDataDTO(
+        pathology="dementia",
+        datasets=["demo_data"],
+        filters=None,
```

Comment:
As i am creating the filters implementation, i would like to be provide ( if it is  possible) a TODO.
In which there will be a viable filter case so i can test filters in the test_post_request too.